languages/rust/concepts/entry-api/about.md

```diff
@@ -1 +1,15 @@
-TODO: add information on entry-api concept
+The entry API is often the most idiomatic way to manipulate the contents of a map structure. It's especially useful when you need to alter multiple entries in a map structure.
+
+This exercise covered one particular pattern that the entry API is commonly used for, namely using `HashMap`s to store the frequency of elements.
```

Comment:
If this is general documentation, it shouldn't refer to "this exercise", or the work done in that exercise. Instead, we should just walk about various methods to accomplish the goal.

```rust
*map.entry(key).or_default() += 1;
```